lib/twilio/device.ts

```diff
@@ -982,6 +982,9 @@ class Device extends EventEmitter {
     }, options);
 
     const maybeUnsetPreferredUri = () => {
+      if (!this._stream) {
+        return this._log.warn('UnsetPreferredUri called without a stream');
```

Comment:
```suggestion
        this._log.warn('UnsetPreferredUri called without a stream');
        return;
```
My preference, but I think this makes it more clear and less error-prone just in case the `log` API changes in the future. Very unlikely, but possible.